src/main/java/net/glowstone/block/blocktype/BlockFalling.java

```diff
@@ -56,17 +56,11 @@ protected void transformToFallingEntity(GlowBlock me) {
         }
     }
 
+    /**
+     * @param material A Material
+     * @return true if material will support this block and prevent it from falling or breaking
+     */
     private boolean supportingBlock(Material material) {
```

Comment:
Is it worth just removing this function and replacing the usage with `material.isSolid()`?

Reply:
A similar `supportingBlock` method is defined here: https://github.com/GlowstoneMC/Glowstone/blob/d72e7f0273085796273c3bb9086729fcd935f7ef/src/main/java/net/glowstone/entity/objects/GlowFallingBlock.java#L181
My thought was that keeping the function would clarify the difference between supporting a stationary falling block and a, well, falling falling block.
I think that the name of the function also makes the code a bit clearer and more descriptive.

Tbh there's not much of a reason to keep or remove the function, so let me know what you prefer and it's best we go wit hthat.

Reply:
I agree it makes the code clearer with this function. Nothing actionable